bob/main.py

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
 
-import json
```

Comment:
Json is still used though?

Reply:
No it is not. It was used for the settings.json, but it is not used anymore.

Reply:
Line 60, space command. Also weather command has use of json library?